src/store/selection/types.ts

```diff
@@ -0,0 +1,11 @@
+import { DOMRectMini } from '../../@types';
+
+export type SelectionState = {
+    selection: SelectionItem | null;
+};
+
+export type SelectionItem = {
+    boundingRect: DOMRectInit;
```

Comment:
Is DOMRectInit a built-in type? Can we use it for `rects`, as well?

Reply:
Sorry it should be `DOMRectMini`. `DOMRectInit` is a built-in type but all of its properties are optional. I think `DOMRectMini` is better.

Reply:
Can we use `Required<DOMRectInit>`?